guide/Manhattan_release.rst

```diff
@@ -0,0 +1,423 @@
+   -  ``[stage]_mean.nc``
+   -  ``[stage]_sd.nc``
+   -  ``[stage]_priorinf_{mean,sd}.nc`` (if prior inflation is turned on)
+   -  ``[stage]_postinf_{mean,sd}.nc`` (if posterior inflation is turned on)
+
+   If you are reading and writing ensemble members from a single file, all this information will now be in a single
+   NetCDF file but will be stored in different variables inside that file:
+
+   -  ``[var].nc``
+   -  ``[var]_mean.nc``
+   -  ``[var]_sd.nc``
+   -  ``[var]_priorinf_{mean,sd}.nc`` (if prior inflation is turned on)
+   -  ``[var]_postinf_{mean,sd}.nc`` (if posterior inflation is turned on)
+
+   We also now have options for writing files at four stages of the assimilation cycle:
+   ``'input', 'preassim', 'postassim', 'output'``. This is set in the &filter_nml namelist with stages_to_write.
+
+#. New model_mod.f90 required routines:
+
+   -  ``vert_convert()``
+   -  ``query_vert_localization_coord()``
+   -  ``pert_model_copies()``
+   -  ``read_model_time()``
+   -  ``write_model_time()``
+
+   There are default version of these available to use if you have no special requirements.
+
+#. Several of the model_mod.f90 argument lists have changed
+
+   -  ``model_interpolate()`` now takes in the ``state_handle`` as an argument rather than a state vector array. It also
+      return an array of ``expected_obs`` and ``istatus`` for each of the ensemble members
+   -  ``get_state_meta_data()`` also requires the ``state_handle`` as an argument rather than a state vector array.
+   -  ``nc_write_model_atts()`` has an additional argument ``moel_mod_writes_state_variables``. If true then the
+      model_mod is expected to write out the state variables, if false DART will write out the state variable (this is
+      the prefered method for adding new models, it requires less code from the model developer)
+
+#. There are several namelist changes mainly in the &filter_nml and &perfect_model_mod which are outlined in detail in
+   :doc:`./Manhattan_diffs_from_Lanai`
+
+#. All modules have been moved to *DART/assimilation_code/modules/* directory. And similarly all of the programs have
+   moved to *DART/assimilation_code/programs/*
+
+#. The location modules which were stored in *locations* have moved to *DART/assimilation_code/location* directory
+
+#. The observation converters which were stored in *observations* have moved to *DART/observations/obs_converters*
+   directory
+
+#. The forward operators have moved from *obs_def/obs_def_*_mod.f90* to *observations/forward_operators*
+
+#. The tutorial files have moved to *DART/docs/tutorial directory*
+
+#. The program ``fill_inflation_restart`` is OBSOLETE since DART inflation files are now in NetCDF format. Now inflation
+   files can be filled using ``ncap2``. Here is an example using version 4.4.2 or later of the NCO tools:
+
+   ::
+
+        ncap2 -s "T=1.0;U=1.0;V=1.0" wrfinput_d01 prior_inf.nc'
+        ncap2 -s "T=0.6;U=0.6;V=0.6" wrfinput_d01 prior_sd.nc'
```

Comment:
fill_inflation_restart is still a valid program and is easier to use than the NCO utilities.  yes, you can use ncap2 but there are problems with earlier NCO versions.  plus it's just more clunky and error-prone.   an example of ncap2 is in the program source code, should someone need it sometime.  but how about this as a suggested change here?

```suggestion
#. The program ``fill_inflation_restart`` can be used to create initial inflation restart files for the first assimilation step in a multi-step assimilation.  This allows the scripting to treat the first step the same as subsequent steps for inflation file motion and namelist settings.
```